src/ext/options.js

```diff
@@ -198,6 +198,9 @@ var optionGroups = {
         "Use SVG gutter icons": {
             path: "useSvgGutterIcons"
         },
+        "Show annotations folded lines": {
```

Comment:
Reason I didn't add that is because that pushes `lines` to a new line but I agree that `Show annotations for folded lines` is clearer to read. Not sure what's the right choice here 😄 

<img width="200" alt="Screenshot 2023-04-19 at 13 44 45" src="https://user-images.githubusercontent.com/34573235/233064911-1cf7ab12-8552-4ad2-95ab-f7d05fa47368.png">

Reply:
it's a demo, it doesn't matter how many lines are there in a label. if you want to keep it in a single line, just name it `Annotations for folded lines`, checkbox is informative enough to replace the `show` word